src/pre.l

```diff
@@ -3182,6 +3182,7 @@ static void endCondSection(yyscan_t yyscanner)
   if (state->condStack.empty())
   {
     state->skip=FALSE;
+    warn(g_yyFileName,g_yyLineNr,"the \\endcond does not have a corresponding \\cond in this file");
   }
```

Comment:
Your warning statement should probably read something like:
```
        warn(state->yyFileName,state->yyLineNr,"the ....
```

Reply:
Huh, you were 40 seconds faster!
I tried to describe everything in the issue, linked to the PR.